treescript/tests/test_l10n.py

```diff
@@ -5,6 +5,13 @@
 from scriptworker_client.utils import makedirs
 
 import treescript.l10n as l10n
+from treescript import mercurial
```

Comment:
Hm, do we need this import in this module?

Reply:
We don't. Thanks for catching it!